kiva/tests/drawing_tester.py

```diff
@@ -184,8 +184,8 @@ def save_and_return_dpi(self):
 
         filename = "{0}.png".format(self.filename)
         self.gc.save(filename)
-        image = Image.open(filename)
-        dpi = image.info['dpi']
+        with Image.open(filename) as image:
+            dpi = image.info['dpi']
```

Comment:
Drive-by fix. This caused a `ResourceWarning` when I was running the tests (which I hadn't seen before).